ouroboros-network-framework/src/Ouroboros/Network/Snocket.hs

```diff
@@ -147,19 +150,33 @@ berkeleyAccept ioManager sock = go
           (Socket.close . fst)
           $ \(sock', addr') -> do
             associateWithIOManager ioManager (Right sock')
-            return (Accepted sock' addr', go)
+
+            -- UNIX sockets don't provide a unique endpoint for the remote
+            -- side, but the InboundGovernor/Server requires one in order to
+            -- track connections.
+            -- So to differentiate clients we use a simple counter as the
+            -- remote end's address.
+            --
+            addr'' <- case addr' of
+                           Socket.SockAddrUnix _ ->
+                               return $ Socket.SockAddrUnix $
+                                   "temp-" ++ show cnt
+                           _                     -> return addr'
```

Comment:
It looks strange when it's hanging that much when the previous line is so much to the left; up to you if you want to include it or not.
```suggestion
                           _  -> return addr'
```

Reply:
I prefer to have the `->` aligned.